### PR TITLE
Add logger include directories

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**
In case if logger is not installed to the system SDL should include directories from environment variables

### CODE REVIEW

**Comment by LuxoftAKutsan** [comment]
@agritsevich,, @dev-gh, @dtrunov  please review 

@Abyzhynar @orhan-mehmedov   @okozlovlux  you approoved this pull requiest in #872

**Comment by LuxoftAKutsan** [comment]
Teamcity failed due too configuration error.

File: tools/policy_table_validator/CMakeLists.txt [hunk 1]
```diff
@@ -24,3 +24,7 @@ set (SOURCES
 
 add_executable(policyValidator ${SOURCES})
 target_link_libraries(policyValidator ${LIBRARIES})
+
+if(ENABLE_LOG)
+  target_link_libraries(policyValidator log4cxx -L${LOG4CXX_LIBS_DIRECTORY})
```
Comment (orhan-mehmedov) [inline tools/policy_table_validator/CMakeLists.txt]: @LuxoftAKutsan Isn't it enough just to add Utils to list of ${LIBRARIES}?

Comment (LuxoftAKutsan) [inline tools/policy_table_validator/CMakeLists.txt]: @orhan-mehmedov thanks ! b771096

**Pull request merged**